src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -0,0 +1,31 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.command.Subsystem;
+import frc.robot.commands.AutoClimb;
+
+/**
+ * Climber as a subsystem will be calling the
+ * AutoClimb command as to collect input from
+ * sensors on toaster to move rollers on
+ * Charles
+ *  @author Travis Bettens
+ */
+public class Climber extends Subsystem {
+  // Put methods for controlling this subsystem
+  // here. Call these from Commands.
+
+  @Override
+  public void initDefaultCommand() {
+    // Set the default command for a subsystem here.
+    // setDefaultCommand(new MySpecialCommand());
+    
+    setDefaultCommand(new AutoClimb());
```

Comment:
I think we may have to do this differently, but no worries, for now we can leave autoclimb as the default.